packages/devtools-connect/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "@mongodb-js/tsconfig-devtools/tsconfig.common.json",
   "compilerOptions": {
     "target": "es2020",
-    "lib": ["es2020", "DOM"],
+    "lib": ["es2021", "DOM"],
```

Comment:
this is just cus `AggregateError` was missing in the test type check, I think this is also ok for the shipped code